pkgs/tools/backup/duplicity/default.nix

```diff
@@ -80,51 +88,54 @@ python3.pkgs.buildPythonApplication rec {
     par2cmdline # Add 'par2' to PATH.
   ] ++ lib.optionals stdenv.isLinux [
     util-linux # Add 'setsid' to PATH.
+  ] ++ lib.optionals stdenv.isDarwin [
+    getconf
   ] ++ (with python3.pkgs; [
     lockfile
     mock
```

Comment:
Looking at the diff `future` and `mock` should no longer be necessary:

https://gitlab.com/duplicity/duplicity/-/compare/rel.0.8.23...rel.2.2.3#58ef006ab62b83b4bec5d81fe5b32c3b4c2d1cc2